stdpopsim/genomes.py

```diff
@@ -195,7 +195,10 @@ class Contig:
     interval_list = attr.ib(factory=list)
 
     def __attrs_post_init__(self):
-        self.fully_neutral()
```

Comment:
This was the only place we used `fully_neutral()`, and Contigs come with this by default, so I removed the method.